dashboard/modules/tune/tune_head.py

```diff
@@ -0,0 +1,233 @@
+import logging
+import datetime
+import copy
+import os
+import aiohttp.web
+
+import ray.new_dashboard.modules.tune.tune_consts \
+    as tune_consts
+import ray.new_dashboard.utils as dashboard_utils
+from ray.new_dashboard.utils import async_loop_forever, rest_response
+
+try:
+    from ray.tune import Analysis
+    from tensorboard import program
+except ImportError:
+    Analysis = None
```

Comment:
What happened if the `Analysis` is None?

Reply:
If Analysis is none, the Tune tab availability check will fail, and the tune tab won't be displayed.

Reply:
Thanks. The `self.collect()` does not raise an exception? That will be good.

Reply:
It will not, I added a check.